frontend/src/components/login.scss

```diff
@@ -0,0 +1,48 @@
+.login-wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 150px;
+
+  .border-wrapper {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    border: 6px solid #94BBF5;
+    border-radius: 4px;
+    height: 420px;
+    width: 420px;
+  }
+
+  .logo {
+    margin-top: 64px;
+    width: 128px;
+  }
+
+  .project-name {
+    margin: 8px 0;
+  }
+
+  .moto {
+    margin-top: 0;
+    margin-bottom: 32px;
+    font-weight: normal;
+  }
+
+  input {
+    margin-bottom: 64px;
+    outline: none;
+  }
+
+  input:hover {
+    background:url(../images/google-login-focus.png) no-repeat;
```

Comment:
Could you please clarify the reason to change the image on hover and active states?